TileStache/Goodies/VecTiles/transform.py

```diff
@@ -179,8 +179,11 @@ def building_height(shape, properties, fid, zoom):
     height = _building_calc_height(
         properties.get('height'), properties.get('building:levels'),
         _building_calc_levels)
+    area = properties.get('area')
     if height is not None:
         properties['height'] = height
+        if area is not None:
+            properties['volume'] = height * area
```

Comment:
The changes themselves look good to me, but do you think we should rename the function to be a little bit more general? Perhaps we should fold the `min_height` transform into this as well? I can't really think of a good name to capture this though, eg `building_heights_and_volumes`? Or maybe something even less descriptive, eg `building_3d`? :P